spec/bug_report_templates_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'bug_report_templates' do
+  subject do
+    Bundler.with_clean_env do
+      Dir.chdir(chdir_path) do
+        system({'ACTIVE_ADMIN_PATH' => active_admin_root}, Gem.ruby, template_path)
+      end
+    end
+  end
+
+  let(:active_admin_root) { File.expand_path('../..', __FILE__) }
+  let(:chdir_path) { Pathname.new(active_admin_root).join('lib/bug_report_templates').to_s }
```

Comment:
I think this can be simplified to `File.join(active_admin_root, 'lib', 'bug_report_templates')`